doc/man7/migration_guide.pod

```diff
@@ -467,6 +467,18 @@ Remove your usage of the low level APIs. In this case you will need to rewrite y
 
 =back
 
+=head3 Error code changes
+
+As OpenSSL 3.0 provides a brand new Encoder/Decoder mechanism for working with
+widely used file formats, an application code that checks for particular error
```

Comment:
```suggestion
widely used file formats, application code that checks for particular error
```

Reply:
Fixed, thanks. Could you please reapprove/